source/query/filter.txt

```diff
@@ -486,13 +486,13 @@ How Does the Compass Query Compare to MongoDB and SQL Queries?
 .. toctree::
    :titlesonly:
 
-   /query/project
+   Set Returned Fields </query/project>
    Sort Returned Documents </query/sort>
-   /query/maxtimems
-   /query/collation
-   Skip Through Documents </query/skip>
-   Limit Number of Returned Documents </query/limit>
-   /query-plan
-   /export-query-to-language
-   /query/atlas-search
-   /query/queries
+   Adjust Maximum Time </query/maxtimems>
+   Specify Collation </query/collation>
+   Skip Documents </query/skip>
+   Limit Results </query/limit>
+   View Query Performance </query-plan>
+   Export to Language </export-query-to-language>
```

Comment:
As above, perhaps "Export to a Language"?

Reply:
Done!